code/modules/reagents/chemistry/machinery/pandemic.dm

```diff
@@ -57,7 +57,7 @@
 		if(istype(D, /datum/disease/advance))
 			var/datum/disease/advance/A = D
 			var/disease_name = SSdisease.get_disease_name(A.GetDiseaseID())
-			if(disease_name == "Unknown" && A.mutable)
+			if((disease_name == "Unknown") && A.mutable)
```

Comment:
== already has precedence here.

Reply:
it does look cleaner, though

Reply:
I did this for clarity, I am surprised I had not initially.